### PR TITLE
[4.0] Module edit layouts

### PR DESCRIPTION
**Pull request by brianteeman**
Render all the fields in the sidebar using the layout. There should be no visible difference or any difference in the generated code.

### CODE REVIEW

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 067c33993bfdce102e5a3fd02201f794c3d62b0e<br /><br />No visual changes. Code shows differences but they are expected, e.g. `<fieldset class="form-vertical form-no-margin">` missing after the patch because removed with that PR. The code of the fields themselves is same as without the patch. @brianteeman maybe you should slighty modify the testing instructions.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25330">issues.joomla.org/tracker/joomla-cms/25330</a>.</sub>

**Comment by brianteeman** [comment]
The div with the class is not needed as the classes are already present on the fieldset

**Comment by brianteeman** [comment]
Updated branch to retrigger drone

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 1c9ca5ddba564f2098741ce5c910e5dfea286b59<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25330">issues.joomla.org/tracker/joomla-cms/25330</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25330">issues.joomla.org/tracker/joomla-cms/25330</a>.</sub>

File: administrator/components/com_modules/tmpl/module/edit.php [hunk 1]
```diff
@@ -132,20 +132,11 @@
 			<div class="col-md-3">
 				<div class="card card-light">
 					<div class="card-body">
-						<fieldset class="form-vertical form-no-margin">
```
Comment (wilsonge) [inline administrator/components/com_modules/tmpl/module/edit.php:135]: This is going to merge the two fieldsets together. is there any a11y implications of that? just checking

Comment (brianteeman) [reply administrator/components/com_modules/tmpl/module/edit.php:135]: No its fine - they are all related fields

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**